src/app/info/EditWikiItemView.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { DbWikiRow } from "@/databaseUtils";
+import { WikiEditModeButton, WikiItemAccordionSurface } from "@/app/info/StyleComponents";
+import { TextField } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+import CancelIcon from "@mui/icons-material/Cancel";
+import SaveAltIcon from "@mui/icons-material/SaveAlt";
+import supabase from "@/supabaseClient";
+import { logErrorReturnLogId } from "@/logger/logger";
+
+interface EditViewProps {
+    rowRef?: DbWikiRow;
+    setRowRef: (row?: DbWikiRow) => void;
+    setEditModeRef: (editMode: boolean) => void;
+}
+
+const deleteWikiItem = async (
+    setEditModeRef: (editMode: boolean) => void,
+    setRowRef: (row?: DbWikiRow) => void,
+    rowRef?: DbWikiRow
+): Promise<void> => {
+    if (rowRef) {
+        const result = confirm("Confirm deletion of this item?");
+        if (result) {
+            const response = await supabase
+                .from("wiki")
+                .delete()
+                .match({ wiki_key: rowRef.wiki_key });
+            if (response.error) {
+                logErrorReturnLogId("error fetching wiki data", response.error);
```

Comment:
could you change the audit log message here to be about deleting an item from wiki rather than fetching data?